schema/reference-chart_3-10-0.json

```diff
@@ -0,0 +1,495 @@
+            "type": "array",
+            "items": {}
+          }
+        }
+      },
+      "containers": {
+        "type": "array",
+        "items": {}
+      },
+      "dbMigrationConfig": {
+        "type": "object",
+        "properties": {
+          "enabled": {
+            "type": "boolean"
+          }
+        }
+      },
+      "envoyproxy": {
+        "type": "object",
+        "properties": {
+          "configMapName": {
+            "type": "string"
+          },
+          "image": {
+            "type": "string"
+          },
+          "resources": {
+            "type": "object",
+            "properties": {
+              "limits": {
+                "type": "object",
+                "properties": {
+                  "cpu": {
+                    "type": "string",
+                    "format": "cpu"
+                  },
+                  "memory": {
+                    "type": "string",
+                    "format": "memory"
+                  }
+                },
+                "required": [
+                    "cpu",
+                    "memory"
+                ]
+              },
+              "requests": {
+                "type": "object",
+                "properties": {
+                  "cpu": {
+                    "type": "string",
+                    "format": "cpu"
+                  },
+                  "memory": {
+                    "type": "string",
+                    "format": "memory"
+                  }
+                }
+              }
+            },
+            "required": [
+              "limits"
+            ]
+          }
+        }
+      },
+      "image": {
+        "type": "object",
+        "properties": {
+          "pullPolicy": {
+            "type": "string"
+          }
+        }
+      },
+      "ingress": {
+        "type": "object",
+        "properties": {
+          "annotations": {
+            "type": "object",
+            "patternproperties": {
+                "[a-zA-Z_]+[0-9.]+!@#%&*+-=:;?/><,.": {
+                    "type": "string"
+                },
+                "[a-zA-Z_]+[0-9.]+!@#%&*+-=:;?/.,><": {
+                    "type": "string"
+                },
+                "[a-zA-Z_]+[0-9.]+!@#&%*+-=:;?/><,.": {
+                    "type": "string"
+                },
+                "[a-zA-Z_]+[0-9.]+!@#&%+*-=:;?/><,.": {
+                  "type": "string"
+                }
+            }
+          },
+          "enabled": {
+            "type": "boolean"
+          },
+          "host": {
+            "type": "string"
+          },
+          "path": {
+            "type": "string"
+          },
+          "tls": {
+            "type": "array",
+            "items": {}
+          }
+        }
+      },
+      "ingressInternal": {
+        "type": "object",
+        "properties": {
+          "annotations": {
+            "type": "object",
+            "patternproperties": {
+                "[a-zA-Z_]+[0-9.]+!@#%&*+-=:;?/><,.": {
```

Comment:
why 4 entries and also there is difference in the order of characters across these 4 patterns